gkeys/etc/gkeys.conf

```diff
@@ -1,11 +1,18 @@
 # Gentoo-keys configuration file
 #
 
+[login_gentoo]
+login_email = gentoo
+passwd = gentoo_passwd
+server = smtp.gmail.com
+
```

Comment:
for an example of how to use config stacking, look at the gkeys-gen code. It stacks a gkeys-gent.conf file onto the gkeys.conf file.  Putting all email configs in a separate file means that the code only loads those values if needed.  That way it is less likely to leak any secrets.